Version 0.2 Complete

G-Henderson opened the pull request:
App now ready for networking. All the features work with offline dummy values.

G-Henderson committed:
Update libs.versions.toml

Updated gradle

G-Henderson committed:
Started working on ServerHelper methods

G-Henderson committed:
Version 0.2 completed

App works with offline values. Ready to have networking inserted.

G-Henderson committed:
Add stock indicator

Added indicator to each item in the basket and each item in the browsing view to display whether or not an item is in stock.

G-Henderson committed:
UI Optimisations

This commit contains some minor UI optimisations.

G-Henderson committed:
Updated Basket Display

The name of the housemate which added each item to the basket is now displayed (rather than just their ID).

deserted-operative reviewed (changes requested):
Looks amazing!

Just a few things:
- Would it be possible to add a label somewhere on the UI to show which user is currently signed in - it wasn't immediately obvious to me whether I was George Clooney or Tom Cruise.
- There's a bug where I (as George Clooney) was able to change Tom Cruise's apple quantity by plus/minus 1, when that shouldn't be possible.
- Would it be possible for an item to be removed from the list when its quantity gets set to zero, when looking in the basket view?

G-Henderson committed:
Added Profile Display

Added a profile image, profile name and sign out button to the settings fragment. All of these are hardcoded and just for display.

G-Henderson committed:
Updated BasketFragment

- It is no longer possible to edit other peoples' item quantities.
- When an item's quantity reaches 0 in the basket, the item disappears.

G-Henderson commented:
I have now implemented these changes.

**1.**  You can view which user is logged in from the 'Settings' fragment in the app. The profile image and name are currently hardcoded and the 'sign out' button is also **not** functional.
**2.**  You are no longer able to increase/decrease the quantities of items added to the order by other people.
**3.** When decreasing the quantity of an item in the basket to 0, the basket fragment reloads, meaning the item will disappear.

deserted-operative reviewed (approved):
Perfect!
Thanks for those changes :)
